google/cloud/bigquery/external_config.py

```diff
@@ -835,8 +835,10 @@ def schema(self):
         See
         https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#ExternalDataConfiguration.FIELDS.schema
         """
-        prop = self._properties.get("schema", {})
-        return [SchemaField.from_api_repr(field) for field in prop.get("fields", [])]
+        # TODO: The typehinting for this needs work. Setting this pragma to temporarily
```

Comment:
Just a gentle reminder to add this to #2132

Reply:
All TODO items in this PR have been added to Issue #2132